apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdAll.json

```diff
@@ -1050,13 +1050,6 @@
             "code" : "performing",
             "display" : "Performing provider"
           } ]
-        },
-        "qualification" : {
```

Comment:
Are all of these supposed to be removed now? Might be nice to update these tests to include the new fields.